profiler-parser/src/main/java/eu/rssw/antlr/profiler/ProfilerSessionVisitor.java

```diff
@@ -13,9 +13,9 @@
 import eu.rssw.antlr.profiler.ProfilerGrammarParser.Module_data_lineContext;
 
 public class ProfilerSessionVisitor extends ProfilerGrammarBaseVisitor<Void> {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a constructor to the class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1258)